src/actions/deploy.ts

```diff
@@ -36,26 +35,30 @@ export async function deploy(
     kubectl,
     trafficSplitMethod
   );
+  core.endGroup();
   core.debug("Deployed manifest files: " + deployedManifestFiles);
-
   // check manifest stability
-  core.info("Checking manifest stability");
+  core.startGroup("Checking manifest stability");
+  core.info('\u001b[1mBold Checking manifest stability');
   const resourceTypes: Resource[] = getResources(
     deployedManifestFiles,
     models.DEPLOYMENT_TYPES.concat([
       KubernetesConstants.DiscoveryAndLoadBalancerResource.SERVICE,
     ])
   );
   await checkManifestStability(kubectl, resourceTypes);
-
-  if (deploymentStrategy == DeploymentStrategy.BLUE_GREEN) {
-    core.info("Routing blue green");
-    const routeStrategy = parseRouteStrategy(
-      core.getInput("route-method", { required: true })
-    );
-    await routeBlueGreen(kubectl, inputManifestFiles, routeStrategy);
+  core.endGroup();
+  
+  if (deploymentStrategy == DeploymentStrategy.BLUE_GREEN) {    
+    core.group("Routing Blue/Green", async () => {
+      core.info("\u001b[1mBold Routing blue green");
```

Comment:
would be worth it to add a helper function that prefixes the font to strings. Would make it clearer + make it easier to change